lib/measured/unit_system.rb

```diff
@@ -25,6 +25,10 @@ def unit?(name)
   end
 
   def to_unit_name(name)
+    to_unit_name!(name) rescue nil
```

Comment:
This should rescue the specific error class that we raise. If there's a `nil` error or some other exception we don't expect we shouldn't silently swallow it. Always be explicit about the exception types we rescue.